packages/pwa-kit-react-sdk/src/ssr/universal/utils.ts

```diff
@@ -32,9 +39,33 @@ export const getAssetUrl = (path: string) => {
         ? // @ts-ignore
           `${window.Progressive.buildOrigin as string}`
         : `${bundleBasePath}/${process.env.BUNDLE_ID || 'development'}/`
+
     return path ? `${publicPath}${path}` : publicPath
 }
 
+// TODO: Once we establish that we have a new @salesforce/pwa-kit-extensibility package, we can move this utility to
+// it as to not have direct references to extensibilty in the sdk. This will also reduce duplicate code.
+/**
+ * Get the URL that should be used to load a static asset from the bundle.
+ *
+ * @param {string} path - relative path from the build directory to the asset
+ * @function
+ * @returns {string}
+ */
+export const getStaticAssetUrl = (path: string, opts: GetAssetUrlOptions) => {
+    const {extensionPackageName = ''} = opts || {}
+
+    /* istanbul ignore next */
+    const publicPath = onClient
+        ? // @ts-ignore
```

Comment:
I guess the type fails because we need to make sure `window.Progressive.buildOrigin` will always exist.

Reply:
I think it's more that the type is implied to be the window type.. and Progressive doesn't exist in that type.. so we would have to do some custom type defining. 

But this is code copied from the `getAssetUrl` function. So that might make it more alright? lol